lib/liquid/combinators/tags/cycle.ex

```diff
@@ -35,6 +35,8 @@ defmodule Liquid.Combinators.Tags.Cycle do
   import NimbleParsec
   alias Liquid.Combinators.{Tag, General}
 
+  @type t :: {:cycle, [cycle_group: String.t(), cycle_values: [LexicalToken.value()]]}
```

Comment:
`cycle` returns a keyword list, not a tuple. `cycle_group` returns a list with a `String.t()` inside